index.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2014, Yahoo Inc. All rights reserved.
```

Comment:
@jconniff was this left at 2014 because the file hasn't changed? Whereas the others are now 2015?